Makefile

```diff
@@ -29,6 +29,11 @@ updatedeps:
 install:
 	go install -ldflags '-X "main.version=dev build $(dev_build_version)"' .
 
+.PHONY: release
+release:
+	$(INSTALLTOOL) get github.com/goreleaser/goreleaser
```

Comment:
The first time I ran this it introduced a bunch of go.mod changes. However, in grpcurl, I saw that any go.mod/go.sum changes are reverted after running `make release`, so I reverted them too. 

I suppose I could add a helper release script like the one in grpcurl, but since I'm just starting with binaries and not docker or homebrew, I thought I would keep it simple for now.

Reply:
See above: it's because you were doing a `go get` instead of install. BTW, there appears to be a bug in this line: it has a second `get` which I think would make the command fail.

Reply:
🤦